CONTRIBUTING.md

```diff
@@ -53,12 +53,52 @@ When this is not possible, (e.g. for data that is no longer available), find as
 
 ### Creating an index
 
-The index should be a json file where the top level keys are the unique track
-ids of the dataset. The values should be a dictionary of files associated with
-the track id, along with their checksums.
+`mirdata`'s structure relies on `json` objects called `indexes`. Indexes contain information about the structure of the 
+dataset which is necessary for the loading and validating functionalities of `mirdata`. In particular, indexes contain 
+information about the files included in the dataset, their location and checksums.
 
+The index is a json file with the mandatory top-level keys `version` and `metadata` and at least one of the optional 
```

Comment:
does `metadata` need to be mandatory? I can imagine some datasets with no separate metadata file.